locale/en/knowledge/advanced/buffers/how-to-use-buffers.md

```diff
@@ -33,22 +33,32 @@ In the wild, buffers are usually seen in the context of binary data coming from
 
 There are a few ways to create new buffers:
 
-    var buffer = new Buffer(8);
+```js
+var buffer = Buffer.alloc(8);
+// This will print out 8 bytes of zero:
+// <Buffer 00 00 00 00 00 00 00 00>
+```
+This buffer is uninitialized and contains 8 bytes of zero, notice that `new Buffer(size)` is deprecated since v6.0.0.
```

Comment:
The buffer is initialized with zeroes. If it were uninitialized, you could not predict what values it contains.

Reply:
Maybe don't mention the constructor at all? It's deprecated in all currently supported versions of Node.js. Include prominent links to the Buffer docs in this document and people can find out about stuff like that there.